### ISSUE TITLE
Throws error in Bolt 3.2.13

### ISSUE THREAD

**Issue by ankedsgn** [graham73may/bolt-related-content#1]
Not a fault of this extension, but is related to https://github.com/bolt/bolt/issues/6691

Error shown is:
```
Uncaught Exception: Twig_Error_Runtime

Twig_Error_Runtime in Template.php line 447:
An exception has been thrown during the rendering of a template ("The given alias 'pages' is not unique in FROM and JOIN clause table. The currently registered aliases are: pages.") in "modules/_related_posts.twig" at line 12. 
```

Extension is unfortunately not usable for this version at this time, just wanted to give a heads up.

**Comment by graham73may** [comment]
Hi anketwokings,

I'm a couple of beers in at this point but it sounds like an issue for bolt/bolt? Or is there something you're doing with the extension in particular that is causing this error? 

I don't use relationships much which may be why I've not got this error yet!

Thanks for giving the extension a go, sorry it's not working ☺️

**Comment by ankedsgn** [comment]
Hi Graham! 
It's not you, it's Bolt. :) @xiaohutai pointed this out to me, because his Relatedbytags extension suffers from the same problem. There has been a change in Bolt, that affects both your and his extension, that can only be fixed from the Bolt/bolt side, apparently.

But thanks for the extension, I hope it gets fixed by the Bolties, because it is exactly what I need right now. :)

**Comment by ankedsgn** [comment]
This is fixed in Bolt 3.3.0 RC1! 👍
